charts/rasa-x/templates/rasa-config-files-configmap.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 data:
   rasa-credentials: |
     rasa:
-      url: http://{{ include "rasa-x.host" . }}:{{ .Values.rasax.port }}/api
+      url: {{ default "HTTP" .Values.rasax.scheme }}://{{ include "rasa-x.host" . }}.{{ .Release.Namespace }}.svc:{{ .Values.rasax.port }}/api
```

Comment:
We don't need `default` here. The `scheme` parameter has a value in the `values.yaml` file.

Reply:
```suggestion
      url: {{  .Values.rasax.scheme }}://{{ include "rasa-x.host" . }}.{{ .Release.Namespace }}.svc:{{ .Values.rasax.port }}/api
```